recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "ocp" %}
+{% set version = "7.5.3" %}
```

Comment:
It feels like the version number should be this:
```suggestion
{% set version = "7.5.3.0" %}
```
Is there a reason for being different?

Reply:
Is 7.5.3.0 a valid version number for a package on conda-forge?

Reply:
They follow PEP-440 which has no limit on the number of segments.

Reply:
+1

Reply:
The version is now 7.5.3.0